src/main/kotlin/org/wycliffeassociates/otter/jvm/app/ui/collectionsgrid/view/CollectionsGrid.kt

```diff
@@ -24,43 +28,64 @@ class CollectionsGrid : Fragment() {
-                            addClass(DefaultStyles.defaultCardButton)
-                            text = messages["openProject"]
-                            graphic = MaterialIconView(MaterialIcon.ARROW_FORWARD, "25px")
-                                    .apply { fill = AppTheme.colors.appRed }
-                            isDisableVisualFocus = true
-                            onMousePressed = EventHandler {
-                                viewModel.selectCollection(item)
+    private fun EventTarget.tabContents() : VBox {
+        return vbox {
+            progressindicator {
+                visibleProperty().bind(viewModel.loadingProperty)
+                managedProperty().bind(visibleProperty())
+                addClass(CollectionGridStyles.contentLoadingProgress)
+            }
+            datagrid(viewModel.children) {
+                vgrow = Priority.ALWAYS
+                hgrow = Priority.ALWAYS
+                isFillWidth = true
+                addClass(AppStyles.workingArea)
+                addClass(CollectionGridStyles.collectionsContainer)
+                cellCache { item ->
+                    card {
+                        addClass(DefaultStyles.defaultCard)
+                        cardfront {
+                            innercard(AppStyles.chapterGraphic()) {
+                                title = item.labelKey.toUpperCase()
+                                bodyText = item.titleKey
+                            }
+                            cardbutton {
+                                addClass(DefaultStyles.defaultCardButton)
+                                text = messages["openProject"]
+                                graphic = MaterialIconView(MaterialIcon.ARROW_FORWARD, "25px")
+                                        .apply { fill = AppTheme.colors.appRed }
+                                isDisableVisualFocus = true
+                                onMousePressed = EventHandler {
+                                    viewModel.selectCollection(item)
+                                }
                             }
                         }
                     }
                 }
             }
         }
     }
+
+    override val root = anchorpane {
+        tabpane {
+            anchorpaneConstraints {
+                topAnchor = 0
+                rightAnchor = 0
+                bottomAnchor = 0
+                leftAnchor = 0
+            }
+            tab("Scripture") {
+                tabContents()
+            }
+            tab("tN") {
+                graphic = AppStyles.tNGraphic()
+//                graphic = MaterialIconView(MaterialIcon.HOME, "15px")
+                tabContents()
+            }
+            tabClosingPolicy = TabPane.TabClosingPolicy.UNAVAILABLE
+            prefHeight = MainScreenStyles.menuBarHeight.value
+            // 9 is emperical number to subtract, + 2 for top border width
+            tabMinHeightProperty().bind(prefHeightProperty().subtract(11))
```

Comment:
This does not make me happy. I want the tabHeaderArea height to be the same as MainScreenStyles.menuBarHeight (the home/profile/settings menu.)